README.md

```diff
@@ -1,2 +1,123 @@
+  --source SOURCE       Define the 'source' field for the alert, to indicate where this alert is
+                        coming from. Can be set with PAGERDUTY_SOURCE env var. Defaults to the
+                        local host's FQDN
+  -s {0,1,2}, --status {0,1,2}
+                        Status of the check. Can be set with PAGERDUTY_STATUS env var. Defaults to
+                        value of check status. Valid values: 0 (Ok), 1 (Warning), 2 (Critical)
+  -t TOKEN, --token TOKEN
+                        The authentication token. Can be set with PAGERDUTY_TOKEN env var.
+```
+
+### Deduplication key
+
+The deduplication key, unlike [the official handler]((https://github.com/sensu/sensu-pagerduty-handler)), does not accept a template string and no interpolation will be performed on the value.  A simple string can be passed in.  If none is defined, the environment variable `PAGERDUTY_DEDUP_KEY` will be looked at for a value.  If that is also unavailable/unset, a dedup key will be generated by looking at the event data and combining the values of the {entity namespace}/{entity name}/{check name} fields.
+
+### Summary
+
+The incident summary (only relevant for incident creation), much like the [Deduplication key](#Deduplication-key) field, supports only a non-template string value.  If none is defined, the environment variable `PAGERDUTY_SUMMARY` will be sourced for a value.  If this is also unavailable, the default value will be generated from the event data, by combining {entity namespace}/{entity name}/{check name} and the {check output}.
+
+### Source
+
+The source field defined for the alert is a simple, non-templated string, as with the [Deduplication key](#Deduplication-key) field.  If none is defined, the environment variable `PAGERDUTY_SOURCE` will be examined for a value.  If this is also unavailable, the default value will be generated by looking up the FQDN of the host executing the handler.
+
+### Status
+
+This field allows one to explicitly define the status of the check for the handler, overriding the value defined in the event.  In the case of a `Warning` (1) or `Critical` (2), an incident will be triggered where as a value of `Ok` (0) will trigger a resolve of an incident.  If this field is not defined, the handler will read the value from the event data in the {check status} field.
+
+### Details
+
+The function of this field is largely the driver for the creation of this module.  The desire was to be able to provide a customized data structure to [PagerDuty](https://pagerduty.com/).  This field supports a couple of options:
+
+  1. A normal string.  Example: `This is some text that matters to me.`
+  2. A string that is JSON encoded (it will be decoded to be sent to [PagerDuty](https://pagerduty.com/).  Example: `"{\"field\": \"value\", \"something\": \"else\"}"`
+  3. A super simplified template reference. Example: `"{{ .check.metadata.annotations }}"`
```

Comment:
Suggested edit as I'm not sure "super" translates well here:
A simplified template reference.